### ISSUE TITLE
Investigate (x,y) Ordering Between RSIM and ImVIZ

### ISSUE THREAD

**Issue by PaulHuwe** [spacetelescope/romanisim#66]
RSIM and ImVIZ are transpositions of each other (x <-> y), resulting in images like this:
![rsim_sca_rot](https://github.com/spacetelescope/romanisim/assets/42071634/5af69aaa-8740-4c67-8ed9-41ce22bc2169)

Which should have a shape more like:
![image (1)](https://github.com/spacetelescope/romanisim/assets/42071634/7903434f-67aa-45a8-97e6-5caa24f4d9af)

We need to investigate if RSIM is using the wrong convention, or if this needs to incorporated on the Imviz side.

**Comment by bmorris3** [comment]
Thanks for the heads up. Generating some catalogs and testing on my end too.

**Comment by bmorris3** [comment]
Still looking into this. I can get sensible SCA orientations if I do a particular set of transformations to each SCA's data, similar to an x orientation flip, y orientation flip, _and_ a transpose. I can't figure out if these are sensible, but the cluster looks better:

<img width="1672" alt="Screen Shot 2023-07-14 at 11 56 39" src="https://github.com/spacetelescope/romanisim/assets/3497584/81c6ab7b-9a8d-4b7c-8135-eaeffd2c2f6a">

**Comment by schlafly** [comment]
To try to make progress here... can we pick a particular star on these images and make sure that imviz is putting the star where it should be?

This file was generated from a catalog.  We should be able to plot the positions of the stars in that catalog on top of this image and see if they match.  e.g., something like:
```
im2 = asdf.open('l2-cluster-smaller-sca7.asdf')['roman']['data'].value
cat = Table.read('cat-270-66-cluster-smaller.fits')
wcs = asdf.open('l2-cluster-smaller-sca7.asdf')['roman']['meta']['wcs']
xx, yy = wcs.world_to_pixel(cat['ra'], cat['dec'])
m = (np.abs(yy - 2048) < 2048) & (np.abs(xx - 2048) < 2048) ; plot(xx[m], yy[m], 'r+')
```
That nicely produces points on the sources in the image.  That tells me that the simulator WCS & images are consistent, so I think this has to be on the imviz side?

It's easy for me to imagine (x, y) <-> (y, x) issues; there's the usual one with (x, y) corresponding to image[y, x].  I claim that I'm following the accepted python / imshow / wcs API conventions there, but if you think otherwise, let me know.  I don't have a picture for how we could make an error like im -> im[::-1, ::-1]; that's much harder.

If it's useful to have these files, I've dropped them on /grp/roman/eschlafly/romanisim/20230420.

**Comment by schlafly** [comment]
I don't think that anyone thinks that the romanisim images are wrong in this regard, so I'm closing.
